src/test/java/com/fontys/api/mockrepositories/MockJPARepository.java

```diff
@@ -0,0 +1,489 @@
+package com.fontys.api.mockrepositories;
```

Comment:
Bit big?

Reply:
It's as small as it's going to be. It's mostly code that is from the JPARepository itself.